File: Codeforces/1530/A.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t, n;
    cin >> t;
    while (t--) {
        cin >> n;
        int ans = 1;
        while (n > 0) {
            ans = max(ans, n % 10);
            n /= 10;
        }
        cout << ans << "\n";
    }
    return 0;
}

File: Codeforces/1530/A.kt
import kotlin.math.max

private fun readLn() = readLine() ?: ""
private fun readLns() = readLn().split(" ")
private fun readInt() = readLn().toInt()

fun main() {
    val T = readInt()
    repeat(T) {
        var N = readInt()
        var ans = 1
        while (N > 0) {
            ans = max(ans, N % 10)
            N /= 10
        }
        println(ans)
    }
}